### PR TITLE
TF-3101 Hotfix before release

### PR DESCRIPTION
**Pull request by tddang-linagora**
## Issue

#3101 

## Demo
- Should not call `PublicAsset/set destroy` if there's nothing to destroy

https://github.com/user-attachments/assets/141ee8db-6ba4-4d7d-9ae3-0b5363e53cf8

- Should save the status of default identity

https://github.com/user-attachments/assets/df1f04ca-84b9-4389-9738-5ca719acece2

### CODE REVIEW

**Comment by dab246** [comment]
- [ ] Reference Public Asset image can not remove when click save identity in Edit Identity Viewer


https://github.com/user-attachments/assets/7394b93f-4c6a-4180-b104-ba43d401b043

**Comment by tddang-linagora** [comment]
>Reference Public Asset image can not remove when click save identity in Edit Identity Viewer

Fixed

https://github.com/user-attachments/assets/20918b7e-d5c8-4679-ace2-80101f130923

**Comment by tddang-linagora** [comment]
>Loading icon should be placed in the middle of the signature input box.

Fixed
- Web

https://github.com/user-attachments/assets/6f275f21-b340-4618-93e8-feecbb571b6e


- Mobile

https://github.com/user-attachments/assets/c6bddac7-991b-4e65-afcd-82de6d92bb64

**Comment by dab246** [comment]
> > Loading icon should be placed in the middle of the signature input box.
> 
> Fixed
> 
> * Web
> 
>  Screen.Recording.2024-08-22.at.16.43.56.mov 
> * Mobile
> 
>  Screen.Recording.2024-08-22.at.16.40.06.mov

IMO, the Loading Bar should be in the center of the red frame, as shown below.

Screenshot 2024-08-23 at 11 44 48

**Comment by tddang-linagora** [comment]
>IMO, the Loading Bar should be in the center of the red frame

Done
- Web

https://github.com/user-attachments/assets/db814971-f00c-487b-9dc7-47afee672c3e

- Mobile

https://github.com/user-attachments/assets/5033b171-b5a6-46a8-b9d2-cf925770ac6a

File: lib/features/identity_creator/presentation/identity_creator_view.dart [hunk 1]
```diff
@@ -42,129 +42,139 @@ class IdentityCreatorView extends GetWidget<IdentityCreatorController>
...
+              scrollInfo.metrics.pixels == scrollInfo.metrics.maxScrollExtent) {
+            controller.clearFocusEditor(context);
+          }
+          return false;
+        },
+        child: bodyCreatorView,
+      );
+    }
+
     if (PlatformInfo.isWeb) {
```
Comment (dab246) [inline lib/features/identity_creator/presentation/identity_creator_view.dart:178]: please combine condition

Comment (hoangdat) [reply lib/features/identity_creator/presentation/identity_creator_view.dart]: fixed, please check

**Review by dab246 — APPROVED** [review]
lgtm

**Review by hoangdat — APPROVED** [review]

**Comment by dab246** [comment]
- [ ] The signature message input box is too big. https://github.com/linagora/tmail-flutter/issues/3101#issuecomment-2306298330

**Pull request merged**